server/services/RunService.js

```diff
@@ -393,7 +410,27 @@ async function sequelizeRunsToJsonRuns(sequelizeRuns) {
             run_status_id: run.run_status_id,
             run_status: run.RunStatus.name,
             test_version_id: run.test_version_id,
-            testers: run.Users.map(u => u.id)
+            testers: run.Users.map(u => u.id),
+            tests: run.ApgExample.Tests.reduce((acc, test) => {
+                acc.push({
+                    id: test.id,
+                    file: test.file,
+                    name: test.name,
+                    execution_order: test.execution_order,
+                    results: test.TestResults.filter(
+                        testResult => testResult.run_id == run.id
```

Comment:
this might take care for that for results? but you still will see a test with no results.